CMakeLists.txt

```diff
@@ -6,11 +6,13 @@ catkin_simple()
 
 include(ExternalProject)
 
+set(YAML_CPP_TAG 11607eb5bf1258641d80f7051e7cf09e317b4746)
```

Comment:
would it be possible to use a tag here? e.g. yaml-cpp-0.5.3
that way it would be easier to figure out the version